components/filter/events.tsx

```diff
@@ -17,6 +16,10 @@ import { EventsRelativeBox } from "../events/eventsRelativeBox";
 import { CITY_MAP } from "../util/constants/country";
 import { sswOrganisation } from "../util/constants/json-ld";
 import { FilterBlock } from "./FilterBlock";
+import {
+  useFetchFutureEvents,
+  useFetchPastEvents,
+} from "../../hooks/useFetchEvents";
```

Comment:
maybe we should import it something like @/hook/*.

Reply:
done - updated to use aliases

Reply:
Reverted these changes - Build was failing probably due to the issues you raised in this issue: https://github.com/tinacms/tinacms/issues/4488